native/src/components/SearchListItem.tsx

```diff
@@ -0,0 +1,102 @@
+import Highlighter from 'react-native-highlight-words'
+import styled, { useTheme } from 'styled-components/native'
+
+import { CategoryModel, getExcerpt, normalizeString } from 'api-client'
+
+import { SEARCH_PREVIEW_MAX_CHARS } from '../constants'
+import { contentDirection } from '../constants/contentDirection'
+import { PageResourceCacheStateType } from '../utils/DataContainer'
+import { getCachedThumbnail } from './Categories'
+import { CategoryThumbnail } from './CategoryListItem'
+import StyledLink from './StyledLink'
+
+const FlexStyledLink = styled(StyledLink)`
+  display: flex;
+  flex-direction: column;
+`
+const DirectionContainer = styled.View<{ language: string }>`
+  display: flex;
+  flex-direction: ${props => contentDirection(props.language)};
+`
+
+const SearchEntryContainer = styled.View`
+  flex: 1;
+  flex-direction: column;
+  align-self: center;
+  padding: 15px 5px;
+  border-bottom-width: 1px;
+  border-bottom-color: ${props => props.theme.colors.themeColor};
+`
+
+const TitleDirectionContainer = styled.View<{ language: string }>`
+  flex-direction: ${props => contentDirection(props.language)};
+  align-items: center;
+`
+
+const HighlighterCategoryTitle = styled(Highlighter)<{ language: string }>`
+  flex-direction: ${props => contentDirection(props.language)};
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+  color: ${props => props.theme.colors.textColor};
+`
+
+type SearchListItemProps = {
+  category: CategoryModel
+  contentWithoutHtml: string
+  resourceCache: PageResourceCacheStateType
+  onItemPress: (category: CategoryModel) => void
+  language: string
+  query: string
+}
+
+const SearchListItem = ({
+  language,
+  category,
+  resourceCache,
+  contentWithoutHtml,
+  onItemPress,
+  query,
+}: SearchListItemProps): ReactElement => {
+  const theme = useTheme()
+  const { title, thumbnail } = category
+  const excerpt = getExcerpt(contentWithoutHtml, { query, maxChars: SEARCH_PREVIEW_MAX_CHARS })
+
+  const Content =
+    query && excerpt.length > 0 ? (
+      <Highlighter
+        searchWords={[query]}
+        sanitize={normalizeString}
+        textToHighlight={excerpt}
+        autoEscape
+        highlightStyle={{ backgroundColor: theme.colors.backgroundColor, fontWeight: 'bold' }}
+      />
+    ) : null
+
+  const Title = (
+    <HighlighterCategoryTitle
+      language={language}
+      autoEscape
+      textToHighlight={title}
+      sanitize={normalizeString}
+      searchWords={query ? [query] : []}
+      highlightStyle={{
+        fontWeight: 'bold',
+      }}
+    />
+  )
+  return (
+    <FlexStyledLink onPress={() => onItemPress(category)} underlayColor={theme.colors.backgroundAccentColor}>
+      <DirectionContainer language={language}>
+        <SearchEntryContainer>
+          <TitleDirectionContainer language={language}>
+            {!!thumbnail && <CategoryThumbnail source={getCachedThumbnail(category, resourceCache)} />}
```

Comment:
i think we shouldn't reuse the thumbnail here since the margin left should be removed. All items should align left
<img width="346" alt="image" src="https://user-images.githubusercontent.com/37902063/232438995-69db7e17-75e1-4ff3-a356-50c987c48e3d.png">
While searching it's getting worser
<img width="340" alt="image" src="https://user-images.githubusercontent.com/37902063/232439350-42b2c717-131b-438d-93d8-e2c467c6612e.png">

Reply:
I fixed this by using only right/left margin as needed. Looks much better now. Thank you for the feedback.